package.json

```diff
@@ -13,5 +13,10 @@
                 }
             }
         }
+    },
+    "// resolutions.semver-max&is-odd": "// these are for the npm/private/test/package.json but pnpm has a bug where resolutions need to be specified in the root package.json but the paths are still relative to the package.json where the resolution applies",
```

Comment:
🤔 

So what if `semver-max` was also used in another location where this relative path doesn't work?

Reply:
Yep. Its an annoying pnpm bug

Reply:
Is there a link to an issue?

Reply:
I didn't look to see if it had been reported yet

Reply:
They have a few of these "it has to be done in the root package.json" issues for other fields besides resolutions